benchmark/results/AMD Ryzen 7 2700X Eight-Core Processor/README.md

```diff
@@ -0,0 +1,182 @@
+object (union, implicit) | 12153.761669412412 | Failed | Failed | Failed | Failed | Failed
+array (recursive) | 4439.148626523559 | 4310.370508285235 | 1447.208402432283 | 279.45205479452056 | 6.079587324981578 | 1.652285661832201
+array (union, explicit) | 2503.776017682814 | 1341.5570175438595 | 485.3182564289623 | 216.46060158700868 | 2.1953896816684964 | 27.05749718151071
+array (union, implicit) | 850.018268176836 | Failed | Failed | Failed | Failed | Failed
+ultimate union | 397.02643171806164 | Failed | Failed | Failed | Failed | Failed
+
+
+
+## assertType (iterate)
+
+![assertType (iterate) benchmark](images/assertType_po_iterate_pc.svg)
+
+ Components | typescript-json | typebox | io-ts | zod | class-validator 
+------------|-----------------|---------|-------|-----|-----------------
+object (simple) | 110274.38370846731 | 1958.0805295091009 | 9580.989876265467 | 2240.67300658376 | 82.20699870394373
+object (hierarchical) | 36145.270270270266 | 468.41349605168693 | 1974.307212332538 | 251.83630640083948 | 27.467496795458707
+object (recursive) | 28299.341486359357 | 217.17171717171715 | 895.8873656909968 | 44.48198198198198 | 18.15431164901664
+object (union, explicit) | 4131.076701476413 | 77.57197872730606 | 588.7694588584136 | 23.229461756373937 | 66.14785992217898
+object (union, implicit) | 3946.308724832215 | Failed | Failed | Failed | Failed
+array (recursive) | 1651.873951146746 | 22.10466654071415 | 83.13452009543036 | 6.1395348837209305 | Failed
+array (union, explicit) | 1531.9740500463392 | 10.563380281690142 | 48.30460314453495 | 2.2140221402214024 | 24.430871737923376
+array (union, implicit) | 625.689084895259 | Failed | Failed | Failed | Failed
+ultimate union | 176.72886937431394 | Failed | Failed | Failed | Failed
+
+
+
+## assertType (throw)
+
+![assertType (throw) benchmark](images/assertType_po_throw_pc.svg)
+
+ Components | typescript-json | typebox | io-ts | zod | class-validator 
+------------|-----------------|---------|-------|-----|-----------------
+object (simple) | 30341.643582640812 | 1672.6179289607217 | 5989.536352155872 | Failed | 91.17432530999271
+object (hierarchical) | 20156.774916013437 | 439.3190554640308 | 1916.212662032688 | 280.79371022089106 | 35.587188612099645
+object (recursive) | 3043.3999639834324 | Failed | Failed | Failed | 54.7845142439737
+object (union, explicit) | 3729.985443959243 | 91.37426900584794 | 679.3734666918285 | 34.560221185415585 | 73.31378299120233
+object (union, implicit) | 3376.9063180827884 | Failed | Failed | Failed | Failed
+array (recursive) | 631.85281546181 | 34.904013961605585 | 74.08779403593257 | 15.067048365225252 | 9.177679882525696
+array (union, explicit) | 297.50836742283377 | 16.611295681063122 | 36.764705882352935 | 9.632055480639568 | 35.448422545196735
+array (union, implicit) | 109.05125408942202 | Failed | Failed | Failed | Failed
+ultimate union | 182.54837531945967 | Failed | Failed | Failed | Failed
```

Comment:
Is that normal that some of the benchmarks are failing?

Reply:
Yes, they all fail in the complicate union case

Reply:
https://github.com/samchon/typescript-json/blob/8744fc177c1eed211e01d96c25440f462caf28f4/benchmark/internal/IsBenchmarker.ts#L54

The code spoils data to intending blocked by validator, but none of them blocks the spoiled data. For reference, I do not check failure case about class-validator, because it always failed to detect complicate error case.